src/executables/uber-time.js

```diff
@@ -8,13 +8,19 @@ import CommandExecutionService from '../services/CommandExecutionService';
 
 const service = new CommandExecutionService();
 
+const address = program.args[0].trim().toString();
```

Comment:
Sorry, I'm probably being dense but I don't entirely follow. Mind explaining how this helps?